File: app/src/main/java/com/example/ryne/myapplication/Java/ProductUtils.java
package com.example.ryne.myapplication.Java;

import java.util.Random;

public class ProductUtils {
    public static double increasePriceItemRandomly(String price) {
        double actualPrice = Double.valueOf(price);
        int a = 80;
        int b = 75;
        int c = new Random().nextBoolean() ? a : b;
        return actualPrice + actualPrice * c / 100;
    }
}

File: app/src/main/java/com/example/ryne/myapplication/Kotlin/ProductUtils.kt
package com.example.ryne.myapplication.Kotlin

import java.util.*

class ProductUtils {
    companion object {
        fun increasePriceItemRandomly(price: String) = when {
            Random().nextBoolean() -> price.toDouble() + price.toDouble() * 80 / 100
            else -> {
                price.toDouble() + price.toDouble() * 75 / 100
            }
        }

        fun isStringNotEmpty(string: String): Boolean {
            return !string.equals("")
        }
    }
}